github/messages.go

```diff
@@ -117,3 +147,42 @@ func validateSignature(signature string, payload, secretKey []byte) error {
 	}
 	return nil
 }
+
+// ParseWebHookType returns the type of payload
+func ParseWebHookType(r *http.Request) string {
+	return r.Header.Get(eventTypeHeader)
+}
+
+// ParseWebHook parses the request into an interface to an activity
+// event type (as returned by Event.Payload())
```

Comment:
This sentence is missing a period at the end, which is inconsistent.

The phrasing could be improved. There's no more request, its parameter is `payload []byte`. No need to mention interface, that's already in the return signature. What it returns is a struct type corresponding to the event type. Taking from `Event.Payload` docs, perhaps something like this:

```
// ParseWebHook parses the event payload. For recognized event types, a value of
// the corresponding struct type will be returned (as returned by Event.Payload()).
```